nx-dev/ui-common/src/lib/headers/header.tsx

```diff
@@ -247,7 +247,7 @@ export function Header(): JSX.Element {
               target="_blank"
               title="Log in to your Nx Cloud Account"
             >
-              <NxCloudIcon className="h-4 w-4" aria-hidden="true" />
+              <NxCloudAnimatedIcon className="h-4 w-4" aria-hidden="true" />
```

Comment:
What is this one doing?

Reply:
We have an animated logo called `NxCloud` so I renamed it to `NxCloudAnimatedIcon` so that the brands page can use the non-animated logo when the page loads.